daffodil-core/src/main/scala/org/apache/daffodil/core/dsom/DFDLDefineVariable.scala

```diff
@@ -101,7 +101,7 @@ class DFDLDefineVariable private (node: Node, doc: SchemaDocument)
       this.SDE("Variables must have primitive type. Type was '%s'.", typeQName.toPrettyString)
     )
 
-  final def createVariableInstance = variableRuntimeData.createVariableInstance
+  final def createVariableInstance = variableRuntimeData.createVariableInstance()
```

Comment:
I was surprised to see that this didn't have test coverage, but sure enough this doesn't get used anywhere and this line can be completely removed.  Don't know if we want to do it in this pull request or not though.

Reply:
I think somewhere else is was commented that we should avoid making changes that aren't related to the 2.13 conversion. Various cleanups and removing dead code probably want to wait until everything is done.